server/data/data-library/src/main/java/org/apache/james/rrt/lib/RecipientRewriteTableManagement.java

```diff
@@ -37,136 +38,136 @@
 public class RecipientRewriteTableManagement extends StandardMBean implements RecipientRewriteTableManagementMBean {
 
     private final RecipientRewriteTable rrt;
-    
+
     @Inject
     protected RecipientRewriteTableManagement(RecipientRewriteTable rrt) throws NotCompliantMBeanException {
         super(RecipientRewriteTableManagementMBean.class);
         this.rrt = rrt;
     }
 
     @Override
-    public void addRegexMapping(String user, String domain, String regex) throws Exception {
+    public void addRegexMapping(String user, String domain, String regex) {
         try {
             rrt.addRegexMapping(user, Domain.of(domain), regex);
         } catch (RecipientRewriteTableException e) {
-            throw new Exception(e.getMessage());
+            throw Throwables.propagate(e);
```

Comment:
I think this is deprecated and we should just `throw new RuntimeException(e)` instead now

Reply:
Isn't it a bit picky, as it is the exact same behaviour of `throw Throwables.propagate(e);`

I must admit I do not understand this remark.